### PR TITLE
add shared workflows for containers

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        type: string
+
+jobs:
+  labeler:
+    permissions:
+      contents: read
+      pull-requests: write
+    name: Labeler
+    runs-on: ubuntu-latest
+    if: github.repository_owner == inputs.allowed_owner
```
Comment (bastelfreak) [inline .github/workflows/labeler.yml:19]: oh I didn't know about inputs.allowed_owner, nice.

Comment (bastelfreak) [reply .github/workflows/labeler.yml:19]: ah derp, I missread it. Does this add any benefit if it's an input field?

Comment (rwaffen) [reply .github/workflows/labeler.yml:19]: i dont know, copied it over from gha-puppet. thought this would be somhow good

Comment (bastelfreak) [reply .github/workflows/labeler.yml:19]: yes, on a shared workflow. we call it like this: https://github.com/voxpupuli/puppet-archive/blob/master/.github/workflows/release.yml#L12-L17
to ensure the action isn't executed when someone forks it. But I think that's only helpful for shared actions/workflows.

Comment (rwaffen) [reply .github/workflows/labeler.yml:19]: this should be shared workflows for container-* projects to use

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**